jsk_arc2017_baxter/euslisp/pick-main.l

```diff
@@ -74,7 +74,7 @@
             (:recognize-object
               '(lambda (userdata)
                  (send *ti* :update-state *arm* :recognize-object)
-                 (send *ti* :recognize-object *arm* :trial-times 3)))
+                 (not (null (send *ti* :recognize-object *arm* :trial-times 3)))))
```

Comment:
`(if (send *ti* :recognize-object *arm* :trial-times 3) t nil)` is easy to understand

Reply:
Thanks. Updated.